pkg/cli/init-templates/cog.yaml

```diff
@@ -0,0 +1,28 @@
+# Configuration for Cog ⚙️
+# Reference: https://github.com/replicate/cog/blob/main/docs/yaml.md
+
+build:
+  # set to true if your model requires a GPU
+  gpu: false
+
+  # a list of ubuntu apt packages to install
+  system_packages:
+    - "libgl1-mesa-glx"
```

Comment:
Might want some indication that these are just example packages. Maybe they should be commented out by default?

Reply:
Yep we waffled on this. You tipped the scales. ⚖️

Reply:
51f59536c135b97daa87676631ebe4b55e911f8e